pkg/ccl/backupccl/generative_split_and_scatter_processor.go

```diff
@@ -72,24 +79,33 @@ func newGenerativeSplitAndScatterProcessor(
 	post *execinfrapb.PostProcessSpec,
 	output execinfra.RowReceiver,
 ) (execinfra.Processor, error) {
-
 	db := flowCtx.Cfg.DB
-	kr, err := MakeKeyRewriterFromRekeys(flowCtx.Codec(), spec.TableRekeys, spec.TenantRekeys,
-		false /* restoreTenantFromStream */)
-	if err != nil {
-		return nil, err
-	}
+	numSplitWorkers := int(spec.NumNodes)
+	numSecondSplitWorkers := 2 * numSplitWorkers
```

Comment:
can we add a comment here about why we chose 2 * numSplitWorkers? I realize this is probably not your doing but a high level reason explaining the factor of 2 would be useful.